### DISCUSSION TITLE
Subscribe to timeseries and get entries

### DISCUSSION THREAD

**Discussion by theresaharbom** [ravendb/ravendb#17303] [Q&A] [answered]
Hi!

Is it possible to subscribe to timeseries change and get the last timeseries entry?

From what I can see, subscribing to timeseries with the Changes API just gets the type of change and the id of the document that the timeseries belong to, but in order to get the timeseries entry that cause the change you need to load the document.

And data subscriptions can only be used to subscribe to changes in a collection (i.e document changes) ?

Is this correct or is there a possibility that I have not found yet?

**Comment by garayx** [comment, marked as answer]
Hi

in subscription there is an option to include the document latest timeseries, but the subscription subscribes to document changes, so you might want to change the document as well when adding timeseries to it.

https://github.com/ravendb/ravendb/blob/65f8960b0426ac80f58a98c79270cf3bc95390df/test/SlowTests/Client/Subscriptions/SubscriptionsBasic.cs#L1024

**Reply by garayx** [reply]
Also we have a ticket opened to implement timeseries subscription
https://issues.hibernatingrhinos.com/issue/RavenDB-15986/Time-Series-subscriptions

**Reply by theresaharbom** [reply]
Thank you for your answer! Updating the document is not option as the document has several timeseries and I need to know which one has a new entry. Will implement a workaround for now and hope that timeseries subscriptions will be implemented in the future.
